home/lcd.asm

```diff
@@ -58,11 +58,11 @@ DisableLCD:: ; 568
 .wait
 ; Wait until VBlank would normally happen
 	ld a, [rLY]
-	cp 145
+	cp LY_VBLANK + 1
 	jr nz, .wait
 
 	ld a, [rLCDC]
-	and %01111111 ; lcd enable off
+	and $ff ^ 1 << rLCDC_ENABLE
```

Comment:
`$ff ^ (1 << rLCDC_ENABLE)` is how similar lines are formatted.

Reply:
Risking enraging the macro haters, I'm tempted to do `NOT EQUS "$ff ^ ", since `~` cannot be used. (Or just do the right thing and report the bug to rgbds.)